build.sbt

```diff
@@ -37,14 +37,14 @@ val root = project.in(file(".")).settings(
   mimaFailOnNoPrevious := false,
   publish / skip := true,
 )
-aggregateProjects(core, sbtplugin, functionalTests)
+aggregateProjects(core.jvm, core.native, sbtplugin, functionalTests)
```

Comment:
@dwijnand this is the new native sub-project. It's essentially the same if you added a new (JVM) module.

Does that answer your question?

Reply:
So it's a just another jar submitted to Sonatype?  And there are no local system requirements?  Compilers, libraries, binaries on PATH, etc?  No platform, i.e. Mac/Linux/Windows, implications?  I'm not thinking about corner cases and bugs, just regular "what adding Scala Native means to library releases" questions.

Reply:
Ah, yup, it's just another Jar :) it's filled with "NIR" (native intermediate representation) files that downstream projects can use to build native binaries. But there's nothing platform-specific we need to do here—we're not shipping a binary.